### DISCUSSION TITLE
[question]: convert lat & lng back to GTA map coords

### DISCUSSION THREAD

**Discussion by casperiv0** [TGRHavoc/live_map#63] [Q&A]
Hi! 

I'm trying to add blips to the in-game map but need to convert the `lat` and `lng` back to the GTAV map. Is there a way how todo that? 

(When someone moves a blip on the live map, it will move it in-game)

Thanks!

**Comment by TGRHavoc** [comment]
Yeah. There is a way but, its not something I'm going to implement.
The map isn't supposed to send data back to the server, its not designed that way. There's no security. If its allowed it can, and probably will, be abused.  

This is one of the reasons I was looking forward to implementing sse. It would much better suite this resource. But, alas. Not possible in fivem :(

**Comment by casperiv0** [comment]
Alright, no worries. Thanks for your reply!

**Comment by TGRHavoc** [comment]
No problem. Is it alright if I close this issue now?

**Comment by casperiv0** [comment]
Yeah, feel free to close it
